README.md

```diff
@@ -60,6 +60,17 @@ For this we need to be able to launch Docker containers inside our Galaxy Docker
 
 The port 8800 is the proxy port that is used to handle Interactive Environments. ``--privileged`` is needed to start docker containers inside docker.
 
+Using passive mode FTP
+----------------------
+
+By default, FTP servers running inside of docker containers are not accessible via passive mode FTP, due to not being able to expose extra ports. To circumvent this, you can use the `--net=host` option to allow Docker to directly open ports on the host server:
+
+  ```bash
+  docker run -d --net=host -v /home/user/galaxy_storage/:/export/ bgruening/galaxy-stable
+  ```
+
+Note that there is no need to specifically bind individual ports (e.g., `-p 80:80`).
```

Comment:
Probably, since if people think `-p 8080:80` will work then they'll be surprised.